modules/light-clients/08-wasm/types/config.go

```diff
@@ -26,10 +28,12 @@ type WasmConfig struct {
 	ContractDebugMode bool
 }
 
-// DefaultWasmConfig returns the default settings for WasmConfig
-func DefaultWasmConfig() WasmConfig {
+// DefaultWasmConfig returns the default settings for WasmConfig.
+// homePath is the path to the directory where the data directory for
+// Wasm blobs and caches will be stored.
```

Comment:
nit: start of sentence with lowercase letter, we can add a classic "The" :D 

```suggestion
// The homePath is the path to the directory where the data directory for
// Wasm blobs and caches will be stored.
```